indico/modules/rb/models/rooms.py

```diff
@@ -605,7 +608,7 @@ def find_with_filters(filters, user=None):
 
         if filters.get('available', -1) != -1:
             repetition = RepeatMapping.convert_legacy_repeatability(ast.literal_eval(filters['repeatability']))
-            is_available = Room.filter_available(filters['start_dt'], filters['end_dt'], repetition,
+            is_available = Room.filter_available(filters['start_dt'], filters['end_dt'], repetition, True,
```

Comment:
i'd pass `include_blockings` as a kwarg; that way it's clearer what it is without checking the function signature